charts/astronomer/values.yaml

```diff
@@ -82,6 +82,7 @@ houston:
     # host: ~
     # port: ~
     # db: ~
+    # connectionLimit: ~
```

Comment:
`backendSecretName` and `backendConnection` are two ways to set the same value. `connectionLimit` isn't part of a connection object, so we may want to pull this out to a sibling to those fields.